### PR TITLE
Change week day to weekday

### PR DESCRIPTION
**Pull request by ErinClaudio**
### Describe your change in plain English.

The extra space in the word "week day" must be removed and changed to "weekday". 
I found the word used in three spaces after using the search feature in VScode. 

### Link to the issue

https://github.com/rubyforgood/pet-rescue/issues/95

### CODE REVIEW

**Review by nsiwnf — APPROVED** [review]
lgtm!

**Comment by ErinClaudio** [comment]
@nsiwnf Could you take one more look at this, please? After I ran a rebase, I had to resolve a merge conflict. The new change in app/views/adopter_profiles/show.html.erb pulls from a .yml file, but a second set of eyes never hurts. Thank you!

File: app/views/adopter_profiles/show.html.erb [hunk 1]
```diff
@@ -90,7 +90,7 @@
           </div>
           <div class="row mb-4 bigger">
             <p class="col-lg-4">
-              <strong><%= t('.hours_alone_on_weekdays') %></strong>
+              <strong>Hours pet will be alone on weekdays</strong>
```
Comment (nsiwnf) [inline app/views/adopter_profiles/show.html.erb:93]: This can be left as `t('.hours_alone_on_weekdays')` - we just want to update the corresponding text in `en.yml`

**Review by nsiwnf — CHANGES_REQUESTED** [review]
One change to update en.yml and rever show.html.erb

File: app/views/adopter_profiles/show.html.erb (hunk 1 above)
Comment (ErinClaudio) [reply app/views/adopter_profiles/show.html.erb:93]: Silly me and thank you

**Review by nsiwnf — APPROVED** [review]
lgtm!
